runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/harness/StreamingWorkerStatusPages.java

```diff
@@ -248,7 +248,7 @@ public interface Builder {
 
     Builder setCurrentActiveCommitBytes(Supplier<Long> currentActiveCommitBytes);
 
-    Builder setWindmillStreamFactory(GrpcWindmillStreamFactory windmillStreamFactory);
+    Builder setWindmillStreamFactory(GrpcWindmillStreams windmillStreamFactory);
```

Comment:
rename method to match class rename?

Reply:
Actually see other comment, think the class shouldn't be renamed instead

Reply:
reverted